### ISSUE TITLE
typo in eq 99 physics manual (octupole)

### ISSUE THREAD

**Issue by rdemaria** [SixTrack/SixTrack#54]
reported from Adrian

from sympy import *
var('x,y',real=True)

x,y=var('x,y',real=True)
px=-re((x+I*y)**3)
py=im((x+I*y)**3)

**Comment by aoeftiger** [comment]
elegant way to prove it ;-)

**Comment by kyrsjo** [comment]
I'm AFK, but maybe @aoeftiger could fork, branch, fix, and pull? That way credit will be where credit is due. 

Great job spotting the doc bug! :D

**Comment by aoeftiger** [comment]
If I weren't writing my thesis I might consider going into the depths of your doc format -- but for the moment I'll leave that to you, hope that is fine.

**Comment by kyrsjo** [comment]
Ah, right.

So, I corrected equation 95 (not 99), see pull request #57. Adrian, could you please confirm that it is now correct?

Also, here is the fixed SymPy code:
```
from sympy import *
x,y=var('x,y',real=True)
px=-re((x+I*y)**3)
px
py=im((x+I*y)**3)
py
```

Best regards,
Kyrre

**Comment by aoeftiger** [comment]
Yes, it's correct now, thanks ;-)

Cheers,
Adrian

**Comment by kyrsjo** [comment]
Fixed in #57
